index.js

```diff
@@ -2,6 +2,7 @@
 
 var fs = require('fs');
 var path = require('path');
+var findUp = require('find-up');
```

Comment:
please use https://github.com/js-cli/node-findup-sync instead. `find-up` is a duplicate library.

Reply:
you should be able to revert changes to `.travis.yml` and node version afterward.

Reply:
I have updated the package and revert my changes to `.travis.yml` and node version.